release-notes/swan-lake-2201.1.0-release-note.md

```diff
@@ -30,9 +30,368 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### New Features
 
+##### The `spread` operator support for `list` constructor
+
+Introduced the `spread` operator support for the `list` constructor expression.
```

Comment:
Why are we emphasizing `spread`? It's not a keyword or a name of a construct in the code.